modules/brute/pop3.yaml

```diff
@@ -0,0 +1,38 @@
+info:
+  name: pop3_brute
+  author: Mrinank Bhowmick
+  severity: 3
+  description: POP3 Bruteforcer
+  reference:
+  profiles:
+    - brute
+    - brute_force
+    - pop3
+
+payloads:
+  - library: pop3
+    steps:
+      - method: pop3_brute_force
+        timeout: 3
+        host: "{target}"
+        ports:
+          - 110
+        usernames:
+          - root
```

Comment:
I think this sould be `- root@{target}` instead, we might need to fix `smtp` and `smtps` as well.